targets/hal/TARGET_Atmel/TARGET_SAM_CortexM0P/TARGET_ADAFRUIT_CP_G18A/PeripheralPins.c

```diff
@@ -58,9 +58,9 @@ const PinMap PinMap_SERCOM_PAD[] = {
     {PA10, SERCOM0_PAD2, 2},
     {PA11, SERCOM0_PAD3, 2},
     {PA12, SERCOM4_PAD0, 3},
```

Comment:
@finneyj @jamesadevine note that PA12-15 should be all on the same SERCOM, and they were not